src/components/ClusterManager.js

```diff
@@ -581,7 +566,7 @@ export default ajaxCaller(class ClusterManager extends PureComponent {
               ` (${Utils.formatUSD(totalCost)} hr)`
             ])
           ]),
-          icon('caretDown', { size: 18 })
+          icon('caretDown', { size: 18, style: { color: isDisabled ? colors.gray[2] : colors.green[0] } })
```

Comment:
I'm not sure if caretDown is the best icon to use here anymore since it implies it'll expand below it.

Reply:
Agreed, changed to gear.